File: vs/Eye.DataModel/DataModel/OwnerModel.cs
﻿using Eye.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Eye.DataModel.DataModel
{
    public class OwnerModel : BaseModel
    {

        //网站样式
        public string ETheme { get; set; }

        //地址
        public string EAddress { get; set; }

        //邮箱
        public string EEmail { get; set; }

        //电话
        public string ETelephone { get; set; }

        //传真
        public string EFax { get; set; }

        //网址
        public string EWebsite { get; set; }

        //
        public string EWechatId { get; set; }

        //
        public string EQQ { get; set; }

        //
        public List<KeyValuePair<string, DateTime>> EOpenHours { get; set; }

    }
}

File: ts/web/src/model/owner.model.ts
import { KeyValuePairs } from "../entity/keyvaluepairs";
import { BaseModel } from "./base.model";

//商家
export class OwnerModel extends BaseModel {

    //网站样式
    ETheme?: string;

    //地址
    EAddress?: string;

    //邮箱
    EEmail?: string;

    //电话
    ETelephone?: string;

    //传真
    EFax?: string;

    //网址
    EWebsite?: string;

    //
    EWechatId?: string;

    //
    EQQ?: string;

    //
    EOpenHours?: KeyValuePairs[];
}